From cedc9df3845dc0d40ff2458f0f7dc6439458f618 Mon Sep 17 00:00:00 2001
From: Lulu Cheng <chenglulu@loongson.cn>
Date: Fri, 25 Oct 2024 06:25:39 +0000
Subject: [PATCH 271/303] LoongArch: Regenerate loongarch.opt.urls.

Fixes: d28ea8e5a704 ("LoongArch: Split loongarch_option_override_internal
                      into smaller procedures")

gcc/ChangeLog:

        * config/loongarch/loongarch.opt.urls: Regenerate.
---
 gcc/config/loongarch/loongarch.opt.urls | 92 +++++++++++++++++++++++++
 1 file changed, 92 insertions(+)
 create mode 100644 gcc/config/loongarch/loongarch.opt.urls

diff --git a/src/gcc/config/loongarch/loongarch.opt.urls b/src/gcc/config/loongarch/loongarch.opt.urls
new file mode 100644
index 000000000..571c504e6
--- /dev/null
+++ b/src/gcc/config/loongarch/loongarch.opt.urls
@@ -0,0 +1,92 @@
+; Autogenerated by regenerate-opt-urls.py from gcc/config/loongarch/loongarch.opt and generated HTML
+
+mfpu=
+UrlSuffix(gcc/LoongArch-Options.html#index-mfpu-2)
+
+msoft-float
+UrlSuffix(gcc/LoongArch-Options.html#index-msoft-float-5)
+
+msingle-float
+UrlSuffix(gcc/LoongArch-Options.html#index-msingle-float)
+
+mdouble-float
+UrlSuffix(gcc/LoongArch-Options.html#index-mdouble-float-1)
+
+msimd=
+UrlSuffix(gcc/LoongArch-Options.html#index-msimd-1)
+
+march=
+UrlSuffix(gcc/LoongArch-Options.html#index-march-7)
+
+mtune=
+UrlSuffix(gcc/LoongArch-Options.html#index-mtune-8)
+
+mabi=
+UrlSuffix(gcc/LoongArch-Options.html#index-mabi-2)
+
+mbranch-cost=
+UrlSuffix(gcc/LoongArch-Options.html#index-mbranch-cost-2)
+
+mcheck-zero-division
+UrlSuffix(gcc/LoongArch-Options.html#index-mcheck-zero-division)
+
+mcond-move-int
+UrlSuffix(gcc/LoongArch-Options.html#index-mcond-move-int)
+
+mcond-move-float
+UrlSuffix(gcc/LoongArch-Options.html#index-mcond-move-float)
+
+mmemcpy
+UrlSuffix(gcc/LoongArch-Options.html#index-mmemcpy)
+
+mstrict-align
+UrlSuffix(gcc/LoongArch-Options.html#index-mstrict-align-1)
+
+mmax-inline-memcpy-size=
+UrlSuffix(gcc/LoongArch-Options.html#index-mmax-inline-memcpy-size)
+
+mexplicit-relocs=
+UrlSuffix(gcc/LoongArch-Options.html#index-mexplicit-relocs-1)
+
+mexplicit-relocs
+UrlSuffix(gcc/LoongArch-Options.html#index-mexplicit-relocs-1)
+
+mrecip=
+UrlSuffix(gcc/LoongArch-Options.html#index-mrecip)
+
+mrecip
+UrlSuffix(gcc/LoongArch-Options.html#index-mrecip)
+
+mcmodel=
+UrlSuffix(gcc/LoongArch-Options.html#index-mcmodel_003d-1)
+
+mdirect-extern-access
+UrlSuffix(gcc/LoongArch-Options.html#index-mdirect-extern-access)
+
+mrelax
+UrlSuffix(gcc/LoongArch-Options.html#index-mrelax-2)
+
+mpass-mrelax-to-as
+UrlSuffix(gcc/LoongArch-Options.html#index-mpass-mrelax-to-as)
+
+mtls-dialect=
+UrlSuffix(gcc/LoongArch-Options.html#index-mtls-dialect-1)
+
+mannotate-tablejump
+UrlSuffix(gcc/LoongArch-Options.html#index-mannotate-tablejump)
+
+mfrecipe
+UrlSuffix(gcc/LoongArch-Options.html#index-mfrecipe)
+
+mdiv32
+UrlSuffix(gcc/LoongArch-Options.html#index-mdiv32)
+
+mlam-bh
+UrlSuffix(gcc/LoongArch-Options.html#index-mlam-bh)
+
+mlamcas
+UrlSuffix(gcc/LoongArch-Options.html#index-mlamcas)
+
+mld-seq-sa
+UrlSuffix(gcc/LoongArch-Options.html#index-mld-seq-sa)
+
-- 
2.33.0

